kd/deployment.yaml

```
@@ -111,11 +111,11 @@ spec:
                   key: rds_password
           resources:
             limits:
-              memory: 32Mi
-              cpu: 400m
+              memory: 64Mi
+              cpu: 600m
```

As we're altering the lines in Git anyway, may as well make them consistent with the rest of the YAML file:
```suggestion
              cpu: 600m
              memory: 64Mi
```

If you accept these changes, make sure you squash before you merge.

---

accepted suggestion and committed.